docs/mint-tooling/assets.md

```
@@ -0,0 +1,116 @@
+
+## Adding an asset
+
+The easiest way is to use the asset manager:
+
+```bash
+npm run asset:add -- path/to/image.png
+```
+
+This copies the file into `src/assets/` and prints the usage snippet you need:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)